.github/workflows/publish.yml

```diff
@@ -0,0 +1,57 @@
+name: Publish packages
+
+# single workflow for both canary and stable as we can only specify a single
+# workflow file for trusted publishing
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 0 * * *' # Daily at midnight UTC
```

Comment:
`on:push:master` will happen on every merge. Why would we need `on:schedule:cron`?

Reply:
Also, this is a lot of noise for contributors, might want to have different approaches for stable/canary where the canary doesn't require accepting.

Reply:
This shouldn't be there, forgot to remove this when copying from `mui-public`